src/plugins/intel_cpu/src/utils/debug_capabilities.cpp

```diff
@@ -18,6 +18,13 @@
 #include "snippets/op/subgraph.hpp"
 #include <ie_ngraph_utils.hpp>
 
+namespace dnnl {
+namespace impl {
+std::ostream &operator<<(std::ostream &ss, const primitive_attr_t *attr);
```

Comment:
the implementation of this two operator seem to be in oneDNN, maybe it will be better that  also put declaration in onednn.

Reply:
they are actually internal functions not exposed by oneDNN in any headers, but I found it's useful for debugging purpose, when you failed to construct some primitive, you can check attr to see if something is wrong, I prefer not to add more patches into oneDNN and just use it silently.